project_event/models/project_task.py

```diff
@@ -215,6 +217,29 @@ def create_new_activity(self, vals):
             .next_by_code('project.task.activity')
         return super(Task, self).create(vals)
 
+    @api.multi
+    def write(self, vals):
+        if self.activity_task_type == 'activity':
+            self.write_task(vals)
+            self.write_activity(vals)
+        elif self.activity_task_type == 'task':
```

Comment:
@jananjoy what if it is not task or activity? need another else

Reply:
@rimbendhaou :Done